pmm-app/tests/ia/alertRules_test.js

```diff
@@ -0,0 +1,162 @@
+const assert = require('assert');
+const page = require('./pages/alertRulesPage');
+
+const rules = new DataTable(['template', 'templateType', 'ruleName', 'threshold', 'duration',
+  'severity', 'filters', 'channels', 'activate']);
+
+for (const i in page.rules) {
```

Comment:
nit: could be better if we use a more modern API like `Object.values(page.rules).forEach(rule => ...)`. This way you don't need to bother with indices :)

Reply:
sure, fixing it. Thanks